### PR TITLE
{vis}[intel/2018a] scikit-image v0.14.0 w/ Python 3.6.4

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2649.swalot.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/6ea2f003546e75b49b6e5c97726e168c for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2007.delcatty.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/d81a9da4574926a3948d79e800e1ab71 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3164.skitty.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/17d3bd85fd063483145a448f4a54b827 for a full test report.

File: easybuild/easyconfigs/s/scikit-image/scikit-image-0.14.0-intel-2018a-Python-3.6.4.eb [hunk 1]
```diff
@@ -0,0 +1,49 @@
+easyblock = 'Bundle'
```
Comment (migueldiascosta) [inline easybuild/easyconfigs/s/scikit-image/scikit-image-0.14.0-intel-2018a-Python-3.6.4.eb]: change to `PythonBundle`?

Comment (boegel) [reply easybuild/easyconfigs/s/scikit-image/scikit-image-0.14.0-intel-2018a-Python-3.6.4.eb]: Sure! Done in 0dc1a85c9

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3103.skitty.os - Linux centos linux 7.5.1804, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/7cf14382aad371a0a91c02273543350e for a full test report.

**Comment by migueldiascosta** [comment]
Test report by @migueldiascosta
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
grc-cluster1 - Linux centos 6.10, Intel(R) Xeon(R) CPU E5-2640 0 @ 2.50GHz, Python 2.7.14
See https://gist.github.com/954ff8db6ede12539697bcd9b04e888e for a full test report.

**Comment by migueldiascosta** [comment]
Going in, thanks @boegel!

**Pull request merged**